src/commands.ts

```diff
@@ -628,4 +630,50 @@ export class Commands {
     this.telemetry.sendEvent('createStripeSample');
     await stripeSamples.selectAndCloneSample();
   };
+
+  createWebhookEndpoint = async (
+    stripeDaemon: StripeDaemon,
+    stripeOutputChannel: vscode.OutputChannel,
+    stripeWebhooksViewProvider: StripeWebhooksViewProvider,
+  ) => {
+    const url = await vscode.window.showInputBox({
+      prompt: 'URL of the webhook endpoint.',
+      value: 'https://',
+    });
+    const description = await vscode.window.showInputBox({
+      prompt: 'Optional description of what the webhook is used for.',
+    });
+    const connect = await vscode.window.showInputBox({
+      prompt: 'Should this endpoint receive events from connected accounts (`y`), or from your account (`n`).',
+    });
+    const isConnect = connect === 'y';
+
+    const createRequest = new WebhookEndpointCreateRequest();
+    createRequest.setUrl(url || '');
+    createRequest.setDescription(description || '');
+    createRequest.setConnect(isConnect);
+
+    const daemonClient = await stripeDaemon.setupClient();
+    daemonClient.webhookEndpointCreate(createRequest, (error, response) => {
+      if (error) {
+        if (error.code === 12) {
+          // https://grpc.github.io/grpc/core/md_doc_statuscodes.html
+          // 12: UNIMPLEMENTED
```

Comment:
The grpc package has constants for these: https://github.com/grpc/grpc-node/blob/c1d68a00aceb552a49ffab1bba502896fc9d1b46/packages/grpc-js/src/constants.ts#L31

Reply:
weird... i remember i fixed all the error code in another PR........

Reply:
oh nvm... it was in the intergration insight PR that i reverted....